packages/next-templates/src/app/globals.css

```diff
@@ -294,9 +307,11 @@ utrecht-alert
   inline-size: 100%;
 }
 
-.voorbeeld-section-container-wrapper {
+.voorbeeld-button-group {
```

Comment:
Zouden deze styles niet beter kunnen op `example-toptask-card__icon`?

Reply:
ButtonGroup gaat over de Heading2 en Button in de tweede sectie.